sdks/python/apache_beam/yaml/json_utils.py

```diff
@@ -172,26 +172,34 @@ def json_to_row(beam_type: schema_pb2.FieldType) -> Callable[[Any], Any]:
     return lambda value: value
   elif type_info == "array_type":
     element_converter = json_to_row(beam_type.array_type.element_type)
-    return lambda value: [element_converter(e) for e in value]
+    return lambda value: [
+        element_converter(e) for e in value
```

Comment:
Rather than modify each of these cases, I would think that element_converter should accept None for nullable types (and reject it otherwise).